onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -0,0 +1,285 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#include <inference_engine.hpp>
+// IE defines a macro 'OPTIONAL' that conflicts the remaining headers using MSVC
```

Comment:
actually, the macro is from a windows header.
see https://github.com/onnx/onnx/issues/1065
it was recently fixed in ONNX, 
https://github.com/onnx/onnx/commit/da13be2d8494b99807bd1f071b40c40cc6b6d1cd
so this workaround should no longer be needed.